stable/appmesh-controller/values.yaml

```diff
@@ -40,3 +40,7 @@ rbac:
   create: true
   # rbac.pspEnabled: `true` if PodSecurityPolicy resources should be created
   pspEnabled: false
+
+logging:
+  #logging.verbosity: <=3 (INFO), >=4 (DEBUG)
+  verbosity: 0
```

Comment:
Are the verbosity numbers actually related to string level descriptions (i.e. 3 is INFO, 4 is DEBUG), or are these just loose translations? If the former, it might be clearer to allow setting the log level by the string description.

Reply:
This is because of v-style logging in Kubernetes ecosystem. See https://godoc.org/github.com/kubernetes/klog. I can change it but that means we need to be consistent in using V(n) in log statements.

Reply:
I will make this change as users will not know what number to set. Thanks